libs/entitlement/src/main/java/org/elasticsearch/entitlement/runtime/policy/PolicyParser.java

```diff
@@ -67,18 +70,23 @@ protected Scope parseScope(String scopeName) throws IOException {
             }
             List<Entitlement> entitlements = new ArrayList<>();
             while (policyParser.nextToken() != XContentParser.Token.END_ARRAY) {
-                if (policyParser.currentToken() != XContentParser.Token.START_OBJECT) {
+                if (policyParser.currentToken() == XContentParser.Token.VALUE_STRING) {
+                    String entitlementType = policyParser.text();
+                    Entitlement entitlement = parseEntitlement(scopeName, entitlementType);
```

Comment:
It's a little surprising to me that you're able to use the same `parseEntitlement` method both for simple flags (where it's just the flag name) and for structured entitlements like file and network. Looks like the difference is whether the constructor has any arguments. Makes sense, I think!

Reply:
Yes - a ctor with no argument can be an empty object (in YAML), but I wanted it to be cleaner -- so now it can be just a string value.
Before you had to do 
```
module:
   - create_classloader:
 ```
Now you can do 
```
module:
   - create_classloader
 ```
Which I think is slightly cleaner